lisa/util/shell.py

```diff
@@ -708,7 +712,8 @@ def stat(self, path: PurePath) -> os.stat_result:
         Outputs:
             os.stat_result: The status structure/class
         """
-        assert isinstance(path, Path), f"actual: {type(path)}"
+        path = Path(path)
+        # assert isinstance(path, Path), f"actual: {type(path)}"
```

Comment:
Just remove the commented line. The git can keep history.

Reply:
sure, i will do it.